backend/src/DTO/tags.model.ts

```diff
@@ -6,6 +6,7 @@ export interface subDefaultTag {
   login: string;
   content: string;
   superContent: string;
+  visibility: string;
```

Comment:
```suggestion
  visibility: "public" | "private";
```
두가지 문자열 중 하나만 보낸다는 점을 명시해주는 게 더 좋을 것 같습니다!